tools/utils/fs.ts

```diff
@@ -12,6 +14,10 @@ export function readJson(path: string) {
   return JSON.parse(readFileSync(path).toString());
 }
 
-export function writeJson(path: string, object) {
+export function writeJson(path: string, object: any) {
   return writeFileSync(path, JSON.stringify(object, null, 2));
 }
+
+export function readWorkspaceJson() {
+  return new Workspaces(join(__dirname, '../../')).readWorkspaceConfiguration();
```

Comment:
Use nrwl provided class workspaces to ensure we always read worskpace json in same manner as nrwl/nx :smile: